code/datums/outfit.dm

```diff
@@ -220,6 +220,12 @@
 	if(socks)
 		user.socks = initial(socks.name)
 
+
+// SKYRAT EDIT ADDITION START - Underwear and bra split
+	if(bra)
+		user.bra = initial(bra.name)
+// SKYRAT EDIT END
```

Comment:
These comments should be following the same level of indentation, just to make things easier.
```suggestion
	// SKYRAT EDIT ADDITION START - Underwear and bra split
	if(bra)
		user.bra = initial(bra.name)
	// SKYRAT EDIT END
```